src/pages/javascript/loops/labeled-statement/index.md

```diff
@@ -3,13 +3,77 @@ title: Labeled Statement
 ---
 ## Labeled Statement
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/loops/labeled-statement/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The **Labeled Statement** is used with the **break** and **continue** statements and serves to identify the statement to which the **break** and **continue** statements apply.
```

Comment:
In general, for the keywords (labeled, break, continue, switch), instead of bold, let's put them in single backticks (\`break\`) to format them as code. You can keep the "Labeled Statement" in this line as bold, though.